### DISCUSSION TITLE
Question Regarding to Preview Information in Final Release

### DISCUSSION THREAD

**Discussion by Fantasydog05** [DLR-VSDC/IEEE-MVC-2023#38] [Q&A] [answered]
Hello,

I’d like to ask if this is correct. In the changelog of the final release, preview horizon is extended to 30 seconds with sampling time of 1 second. The way I understand it is that the time difference between each signal is 1 second. However, the time difference from the simulation seems to be 2.5 seconds, which means the total preview horizon is 2.5(seconds)*30(windows) = 75 seconds.  
![image](https://user-images.githubusercontent.com/87512543/221128394-e67fbec9-b835-4285-b2c6-5e88980d5ee7.png)

Thank you for your reply.

**Comment by imeb** [comment]
Dear @Fantasydog05,
Thanks for your question and raising this point.
I double checked the sampling times, and I have found out that the tracks that were provided by DLR (Munich Tracks 5-10) were sampled with **T_s =0.025s**, whereas the standard cycles FTP, ECE, etc. were sampled with 0.01s. 
Since the preview generation code assumes that T_s is 0.01s for all tracks, you see this difference for Munich Tracks, where the preview horizon as you mentioned is 100\*0.025\*30=75 seconds, and for standard tracks 30 seconds.

**I will bring this matter into our internal discussions and will announce if we are going to make it uniform or keep it as it is.**
Please keep checking the repository for further updates.
Best,
Iman

**Comment by imeb** [comment, marked as answer]
Dear @Fantasydog05,
Munich tracks (track numbers 5-10) are updated here https://github.com/DLR-VSDC/IEEE-MVC-2023/commit/b90441dae094f5fd8a37510fb39edd075b9340bd. In fact, they are resampled with the same sampling time of 10ms same as other tracks, hence the preview horizon for all tracks are now 30 seconds.
Thanks for your attention.
Best,
Iman

**Reply by Fantasydog05** [reply]
Thank you for the modification!
